src/mlpack/methods/det/dt_utils_impl.hpp

```diff
@@ -148,30 +150,27 @@ DTree* mlpack::det::Trainer(arma::mat& dataset,
   std::vector<std::pair<double, double> > prunedSequence;
   while (dtree.SubtreeLeaves() > 1)
   {
-    std::pair<double, double> treeSeq(oldAlpha,
-        dtree.SubtreeLeavesLogNegError());
+    std::pair<double, double> treeSeq(oldAlpha, dtree.SubtreeLeavesLogNegError());
     prunedSequence.push_back(treeSeq);
     oldAlpha = alpha;
     alpha = dtree.PruneAndUpdate(oldAlpha, dataset.n_cols, useVolumeReg);
 
     // Some sanity checks.  It seems that on some datasets, the error does not
     // increase as the tree is pruned but instead stays the same---hence the
     // "<=" in the final assert.
-    Log::Assert((alpha < std::numeric_limits<double>::max()) ||
-        (dtree.SubtreeLeaves() == 1));
+    Log::Assert((alpha < std::numeric_limits<double>::max()) || (dtree.SubtreeLeaves() == 1));
```

Comment:
Yeah---the reason for 80-character lines is in part because it makes much easier to print, or include in a paper, or something like this.  Helpful especially for manuals or PDFs generated by Doxygen, etc..